.circleci/config.yml

```diff
@@ -3,93 +3,58 @@
 # Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
 jobs:
-  build_server:
-    working_directory: /go/src/github.com/aerogear/mobile-developer-console
-
-    docker:
-      - image: circleci/golang:1.10
-
-    steps:
-      - checkout
-      - run: go get github.com/mattn/goveralls
-      - run: go get -u github.com/golang/dep/cmd/dep
-      - run: make build
-      - run: make test_cover
 # Disable goveralls to make build work when merging from forks
 #      - run: /go/bin/goveralls -coverprofile=coverage-all.out -service=circle-ci -repotoken=$COVERALLS_TOKEN
 
   build_ui:
     working_directory: ~/mobile-developer-console
-
     docker:
       - image: circleci/node:8.15.0
 
     steps:
       - checkout
-      - run: make ui-check-code-style
-      - run: make ui-test-cover
-      - run: make ui
-      - persist_to_workspace:
-          root: .
-          paths:
-            - ui/build
+      - run: make check-code-style
+      - run: make test-cover
+      - run: make build
 
 
   docker_push_master:
-    working_directory: /go/src/github.com/aerogear/mobile-developer-console
+    working_directory: ~/mobile-developer-console
     docker:
-      - image: circleci/golang:1.10
+      - image: circleci/node:8.15.0
```

Comment:
Should/could we go straight to Node.js 10?

Reply:
probably should. Will try it.